cookiecutter-django-ida/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:xenial as app
+
+RUN apt update && \
+  apt install -y git-core language-pack-en python3.5 python3-pip python3.5-dev \
+  build-essential libffi-dev libmysqlclient-dev libxml2-dev libxslt-dev libjpeg-dev libssl-dev && \
+  pip3 install nodeenv && \
+  pip3 install --upgrade pip setuptools && \
+  rm -rf /var/lib/apt/lists/*
+
+RUN ln -s /usr/bin/pip3 /usr/bin/pip
+RUN ln -s /usr/bin/python3 /usr/bin/python
+
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+COPY . /edx/app/repo_name
+WORKDIR /edx/app/repo_name
+
+ENV repo_name_CFG /edx/etc/repo_name.yml
```

Comment:
@syed-awais-ali or @jdmulloy I put this WIP PR up, but I haven't figured out how to template out a variable in all caps and have a suspicion that I'm missing some pieces (like the `gunicorn_configuration.py` file)

Reply:
I realized that I didn't need to update the Dockerfile, so this comment no longer applies.